src/js/components/tablero/Tablero.jsx

```diff
@@ -77,18 +82,17 @@ class Tablero extends Component {
   componentDidMount() {
     this.fetchData();
   }
+  componentWillReceiveProps(nextProps) {
```

Comment:
There should be empty line between functions

Reply:
Also why there is no constructor in this component and state had no numberData, which were updated and used before?

Reply:
Constructor added. numberData was used in the state before, but is now mapped in props.

Reply:
I meant that it was not initialized. But it is ok now :)